en/vespa-quick-start.html

```diff
@@ -104,8 +104,7 @@ <h2 id="running-vespa-in-docker">Running Vespa in Docker</h2>
   <div class="pre-parent">
     <button class="d-icon d-duplicate pre-copy-button" onclick="copyPreContent(this)"></button>
 <pre data-test="exec">
-$ vespa clone album-recommendation myapp
-$ cd myapp
+$ vespa clone album-recommendation myapp &amp;&amp; cd myapp
```

Comment:
I wouldn't call this to simplify either - just slightly harder to understand.

Reply:
ditto